### PR TITLE
Feature/logo

### PR DESCRIPTION
**Pull request by DavidDamke**

### CODE REVIEW

File: src/main/java/de/doubleslash/keeptime/view/ViewController.java [hunk 1]
```diff
@@ -156,6 +156,8 @@ private class Delta {
    private ReportController reportController;
 
    private Stage settingsStage;
+
+   private Stage confirmCloseStage;
```
Comment (Death111) [inline src/main/java/de/doubleslash/keeptime/view/ViewController.java:160]: do we need this? does not seem to be used?

File: src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java [hunk 2]
```diff
@@ -260,10 +266,13 @@ private Node createListEntryForProject(final Project p) {
          projectNameLabel.setEffect(null);
       });
 
+      projectNameLabel.setTooltip(new Tooltip(projectNameLabel.getText()));
```
Comment (Death111) [inline src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java:269]: do we need this?

File: src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java [hunk 3]
```diff
@@ -49,6 +53,8 @@
 import javafx.scene.control.MenuItem;
 import javafx.scene.control.MultipleSelectionModel;
 import javafx.scene.control.SelectionMode;
+import javafx.scene.control.Spinner;
+import javafx.scene.control.SpinnerValueFactory.IntegerSpinnerValueFactory;
```
Comment (Death111) [inline src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java:57]: imports can be deleted. (with eclipse "organize imports")

File: src/main/java/de/doubleslash/keeptime/controller/Controller.java [hunk 4]
```diff
@@ -24,6 +24,7 @@
 
 import javax.annotation.PreDestroy;
 
+import javafx.scene.control.ButtonType;
```
Comment (Death111) [inline src/main/java/de/doubleslash/keeptime/controller/Controller.java:27]: can be cleaned up

**Review by Death111 — CHANGES_REQUESTED** [review]

File: src/main/java/de/doubleslash/keeptime/view/ViewController.java (hunk 1 above)
Comment (DavidDamke) [reply src/main/java/de/doubleslash/keeptime/view/ViewController.java:160]: deleted it

File: src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java (hunk 2 above)
Comment (DavidDamke) [reply src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java]: deleted it

File: src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java (hunk 3 above)
Comment (DavidDamke) [reply src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java]: Deleted unused imports

File: src/main/java/de/doubleslash/keeptime/controller/Controller.java (hunk 4 above)
Comment (DavidDamke) [reply src/main/java/de/doubleslash/keeptime/controller/Controller.java]: deleted it

**Comment by Death111** [comment]
@DavidDamke 
- "Add project" and "Edit project" have no icon currently.
- Some popups use the taskbar icon (running time) as icon. e.g. "Confirm Exit", "change with time". Can they also use the app-icon instead?

**Review by Death111 — CHANGES_REQUESTED** [review]
see comment above..

File: src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java [hunk 5]
```diff
@@ -279,7 +270,7 @@ private Node createListEntryForProject(final Project p) {
          alert.setHeaderText("Delete project '" + p.getName() + "'.");
          alert.setContentText(
                "The project will just be hidden from display, as there may be work references to this project.");
-
+         alert.initOwner(mainStage);
```
Comment (Death111) [inline src/main/java/de/doubleslash/keeptime/view/ProjectsListViewController.java:273]: for some popups we have an owner and for some not. can we make it consistent?
I think setting the owner is correct - and then overwrite the icon

File: src/main/java/de/doubleslash/keeptime/common/Resources.java [hunk 6]
```diff
@@ -65,7 +65,9 @@ public enum RESOURCE {
```
Comment (Death111) [inline src/main/java/de/doubleslash/keeptime/common/Resources.java:65]: can you delete the old FXML_ABOUT constant here and with it also the empty about.fxml file?

**Pull request merged**